src/components/SurveyList/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import classNames from 'classnames';
+
+import BlankState from 'components/BlankState';
+
+export const surveyListTestIds = {
+  blankState: 'survey-list__blank-state',
+};
+
+export type SurveyListProps = React.HTMLAttributes<HTMLDivElement>;
+
+const SurveyList = ({ className, ...props }: SurveyListProps) => {
+  const { t } = useTranslation(['survey']);
+
+  return (
+    <div className={classNames('survey-list', className)} {...props}>
+      <BlankState
+        className="survey-list__blank-state"
+        emoji="😎"
```

Comment:
Should we move all used emojis as constant variables?
And maybe we can use all these in `hexadecimal` or `decimal` 🤔
The reason for turning these emoji to code, I think it may be friendly with the code parser 🤔

OR using node package here could help https://www.npmjs.com/package/node-emoji

Reference: https://www.quackit.com/character_sets/emoji/emoji_v3.0/unicode_emoji_v3.0_characters_all.cfm